native/Avalonia.Native/src/OSX/window.mm

```diff
@@ -1234,7 +1250,16 @@ - (void)mouseExited:(NSEvent *)event
 - (void) keyboardEvent: (NSEvent *) event withType: (AvnRawKeyEventType)type
 {
     if([self ignoreUserInput])
+    {
+        auto window = dynamic_cast<WindowImpl*>(_parent.getRaw());
```

Comment:
Since you end up copy-pasting it like this

Reply:
fixed.